modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/DefinitionBuilder.js

```diff
@@ -10,32 +10,16 @@
  */
 
 import React from 'react';
-import ReactFlow, {
-	Background,
-	Controls,
-	ReactFlowProvider,
-} from 'react-flow-renderer';
 
 import '../../css/definition-builder/main.scss';
+import CanvasMode from './canvas-mode/CanvasMode';
 import UpperToolbar from './shared/components/toolbar/UpperToolbar';
 
-const onLoad = (reactFlowInstance) => {
-	reactFlowInstance.fitView();
-};
-
 export default function (props) {
 	return (
 		<div className="definition-builder-app">
 			<UpperToolbar {...props} />
-
-			<div className="definition-canvas">
-				<ReactFlowProvider>
-					<ReactFlow elements={[]} minZoom="0.1" onLoad={onLoad} />
-
-					<Controls showInteractive={false} />
-					<Background size={1} />
-				</ReactFlowProvider>
-			</div>
+			<CanvasMode />
```

Comment:
@jotascript What do you think if we rename it to `CanvasBuilder` and `XMLBuilder` or something similar? Thx